docker-compose.yml

```diff
@@ -5,3 +5,7 @@ services:
     env_file: "config.txt"
     ports:
       - "80:80"
+    volumes:
+      - ~/.aws:/root/.aws:ro
+    environment:
+      - AWS_PROFILE=bedrock_development
```

Comment:
I'd suggest actually naming this `aiproxy-development`, to name it after the role and not one explicit permission that it has. We might, for example, wish to grant S3 or CloudWatch permissions to the user/role.

Reply:
Looks like there was agreement on aiproxy-development. if that's right, @Nokondi would you be up for making this change in a follow-up PR?